app/src/main/java/com/github/se/wanderpals/ui/screens/trip/Dashboard.kt

```diff
@@ -274,33 +276,41 @@ fun Menu(
 fun TopDashboardBar(scope: CoroutineScope, drawerState: DrawerState, tripTitle: String) {
   Column(
       modifier =
-          Modifier.padding(top = 8.dp, start = 8.dp, end = 8.dp).testTag("dashboardTopBar")) {
+          Modifier.background(MaterialTheme.colorScheme.surfaceTint)
+              .padding(top = 8.dp, start = 8.dp, end = 8.dp)
+              .testTag("dashboardTopBar")) {
         Row(
             modifier = Modifier.fillMaxWidth(),
             verticalAlignment =
                 Alignment
                     .CenterVertically // This aligns all children vertically centered in the Row
             ) {
-              ElevatedButton(
-                  modifier = Modifier.testTag("menuButton"),
-                  content = { Icon(Icons.Default.Menu, contentDescription = "Menu") },
-                  onClick = {
-                    scope.launch { drawerState.apply { if (isClosed) open() else close() } }
-                  })
+              Icon(
```

Comment:
You can an IconButton instead

Reply:
I removed the IconButton because it was a bit ugly